docker/docker-compose.yml

```diff
@@ -330,6 +330,14 @@ volumes:
   static-config1:
   static-config2:
   mailbox:
+  tmp_vol:
```

Comment:
Why are these needed?

Reply:
I was under the impression that systemd may require /run and /tmp to be a tmpfs filesystem. I'll try removing these and verify that it still works.

Reply:
To follow up, I took out the tmp_vol and tmp_run volumes, cleared all docker volumes, networks, etc and deployed the stack. I connected to the iml-mailbox container and checked the status of the service and received the following:
```
[root@iml-mailbox /]# systemctl status iml-mailbox
System has not been booted with systemd as init system (PID 1). Can't operate.
Failed to connect to bus: Host is down
```
I then removed the stack and cleared out the volumes, added the tmp volumes back to the docker-compose file and re-deployed the stack. Checking the status of the mailbox service revealed:

```
[root@iml-mailbox /]# systemctl status iml-mailbox
● iml-mailbox.service - IML Mailbox Service
   Loaded: loaded (/etc/systemd/system/iml-mailbox.service; enabled; vendor preset: disabled)
   Active: active (running) since Tue 2020-01-28 14:08:07 UTC; 16s ago
 Main PID: 140 (iml-mailbox)
    Tasks: 1 (limit: 78642)
   Memory: 1.4M
   CGroup: /docker/c0edb9328280642d18a47a6d7618fa5cbe4baf63812038944f7fcf34942e8128/system.slice/iml-mailbox.service
           └─140 /bin/iml-mailbox

Jan 28 14:08:07 iml-mailbox systemd[1]: Started IML Mailbox Service.
Jan 28 14:08:07 iml-mailbox iml-mailbox[140]: Jan 28 14:08:07.748  INFO iml_mailbox: Starting on V4(10.0.46.21:8891)
```